internal/core/software_renderer.rs

```diff
@@ -1398,6 +1409,16 @@ struct GradientCommand {
     bottom_clip: PhysicalLength,
 }
 
+#[derive(Debug)]
+struct ZenoPathCommand {
+    stroke_mask: Option<Vec<u8>>,
```

Comment:
Drive-by thought: Perhaps the `Option` isn't needed here and omitting it would reduce the size of the data structure slightly.

Reply:
Ah, Olivier pointed out that's not the case, so ignore my comment :)